examples/values.rs

```diff
@@ -0,0 +1,6 @@
+extern crate num_cpus;
+
+fn main() {
+    println!("CPUs: {}", num_cpus::get_physical());
```

Comment:
What if we name these "Physical CPUs" and "Logical CPUs"?

Reply:
Good idea, I’ve changed that.